21_microbiome_community.Rmd

```diff
@@ -222,7 +222,31 @@ pheatmap(mat, annotation_row = taxa_clusters,
 ```
 
 In addition to _pheatmap_ package, there are also other packages that
-provide functions for more complex heatmaps, such as [_iheatmapr_](https://docs.ropensci.org/iheatmapr/articles/full_vignettes/iheatmapr.html).
+provide functions for more complex heatmaps, such as [_iheatmapr_](https://docs.ropensci.org/iheatmapr/articles/full_vignettes/iheatmapr.html)
+and 
+[ComplexHeatmap](https://academic.oup.com/bioinformatics/article/32/18/2847/1743594?login=true).
```

Comment:
Now if **sechm** pkg essentially provides `SE` interface to **ComplexHeatmap**, could we remove the **ComplexHeatmap** examples and just stick to **sechm** for simplicity? Unless there is a particular need to keep both.

With **pheatmap** it might be different, if it provides some other important capabilities that are not available through sechm/ComplexHeatmap?

Reply:
Currently, there are no examples with `ComplexHeatmap`

I have only used `pheatmap`, `sechm`, and `ggplot`

We could use here only `sechm`, and move other examples to **Visualization chapter** if we decide to create such

Reply:
Ok that's good.